### PR TITLE
Adds more detailed descriptions to the mda8 variables.

### PR DESCRIPTION
**Pull request by thorbjoernl**
## Change Summary

Adds more detailed descriptions to mda8 variables which references the EU directive. New analyses that makes use of MDA8 calculation will have this new description display in the tool tip on aeroval. 

## Related issue number

https://github.com/metno/AeroToolsIssues/issues/69

## Checklist

* [ ] Start with a draft-PR
* [ ] The PR title is a good summary of the changes
* [ ] PR is set to AeroTools and a tentative milestone
* [ ] Documentation reflects the changes where applicable
* [ ] Tests for the changes exist where applicable
* [ ] Tests pass locally
* [ ] Tests pass on CI
* [ ] At least 1 reviewer is selected
* [ ] Make PR ready to review

### CODE REVIEW

File: pyaerocom/data/variables.ini [hunk 1]
```diff
@@ -3098,11 +3098,10 @@ description=Mass concentration of ozone
 unit = ug m-3
 
 [conco3mda8]
-description=MDA8 of O3 mass concentration
-unit = ug m-3
```
Comment (heikoklein) [inline pyaerocom/data/variables.ini:3102]: Why did you remove the unit?

Comment (thorbjoernl) [reply pyaerocom/data/variables.ini:3102]: Good question. Fixed.

**Review by heikoklein — APPROVED** [review]
